test/special/tslint.js

```diff
@@ -11,36 +11,20 @@ const testCases = [
     expected: [],
   },
   {
-    name: 'detect specific plugins',
+    name: 'skip single built-in config',
```

Comment:
I suppose the second one have covered this. We can remove this one.

Reply:
this is to test that you can use `extends: 'foo'` rather than an array, `extends: ['foo']`. so i think we should leave it here?